src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexReadOperation.cs

```diff
@@ -88,6 +88,8 @@ public override long EntriesCount()
         {
             ExplanationOptions explanationOptions = null;
 
+            // The reason why we keep distincts counts here, is because at the Lucene level there is no sorting unique accesse like we have 
```

Comment:
Should we keep them? @arekpalinski

Reply:
IMO it's okay to leave it

Reply:
This is an explanation on what are the differences. This type of information it is invaluable when you need to debug some issue.